src/document/sharedModel.ts

```diff
@@ -131,6 +131,14 @@ export class GlueSessionSharedModel
     }, false);
   }
 
+  removeTab(name: string): void {
```

Comment:
Probably in a follow up PR, but the viewers of the tab could also be deleted to clean the session file.
Maybe we can open an issue on it.

Reply:
Indeed I wanted to do it in this PR, but it needs more changes in the YDoc structure since we need to keep track of the viewers in a YMap